src/languages/asciidoc.js

```diff
@@ -104,7 +104,13 @@ export default function(hljs) {
         begin: '^(NOTE|TIP|IMPORTANT|WARNING|CAUTION):\\s+',
         relevance: 10
       },
-      // inline strong
+      // inline unconstrained strong
+      {
+        className: 'strong',
+        begin: /\*{2}/,
+        end: /(\n{2}|\*{2})/
+      },
```

Comment:
Also I think we're going to need a variant here for the \n possibility, it's too "wide"... it can match too much, making it not worthy of relevance.
```suggestion
      {
        className: 'strong',
        begin: /\*{2}/,
        variants: [
          { end: /\n{2}/, relevance: 0 },
          { end: /\*{2}/ }
        ]
        
      },
```

Reply:
Hmmm, although now that I think of it `** anything **` is also VERY broad... can these be used without text?  ALl your examples show them surrounding text immediately if so we should add that as a condition here with a look ahead:

```
begin: /\*{2}(?=\w+)/
```

Thoughts?

Reply:
since it's unconstrained, you can have spaces, a single newline...:

```
A sentence ** with words, and more words!  
using bold style :)**.
```

![result](https://user-images.githubusercontent.com/333276/99193435-1b734f80-2779-11eb-9056-198f48d9c9fa.png)

Reply:
Yes, I glanced at your link now and roughly understand they are different.  Here I'm speaking only of auto-detect... we don't just need to know how to highlight Asciidoc but we also have to avoid confusing it with say Javascript... so every rule (that's too generic) needs to have it's relevance tuned or eliminated to avoid false positives.  For example a JS snippet with tons of `**` math formulas might be flagged as Asciidoc... and that should be avoided if possible.

But lets focus on getting them working first I guess. :)